gpytorch/lazy/kronecker_product_added_diag_lazy_tensor.py

```diff
@@ -25,19 +26,23 @@ def __init__(self, *lazy_tensors, preconditioner_override=None):
             raise RuntimeError("One of the LazyTensors input to AddedDiagLazyTensor must be a DiagLazyTensor!")
 
     def inv_quad_logdet(self, inv_quad_rhs=None, logdet=False, reduce_inv_quad=True):
+        # we want to call the standard InvQuadLogDet to easily get the probe vectors and do the
+        # solve but we only want to cache the probe vectors for the backwards
+        with skip_logdet_forward(True):
+            inv_quad_term, func_logdet_term = super().inv_quad_logdet(
+                inv_quad_rhs=inv_quad_rhs, logdet=logdet is not None, reduce_inv_quad=reduce_inv_quad
```

Comment:
```suggestion
                inv_quad_rhs=inv_quad_rhs, logdet=logdet, reduce_inv_quad=reduce_inv_quad
```
As is I think this will always compute the logdet in the forward, since `False is not None`

Reply:
Oops, totally thought that it could be None instead of False.